devops/debian/prerm

```diff
@@ -44,4 +44,9 @@ if [ -d /run/systemd/system ] && [ "$1" = remove ]; then
 fi
 # End automatically added section
 
+# MSFT change: Remove module identity on package removal
```

Comment:
>MSFT [](http://example.com/codeflow?start=2&length=4)

Do we need "MSFT" in comments? MSFT is an internal term that should not appear in our public repo.

We can use "OSConfig Core Team" if needed.

Reply:
This was the pattern we used for any of the changes made after applying the debhelper templates, its not the only place with the "MSFT changes", they are also defined in https://github.com/Azure/azure-osconfig/blob/main/devops/debian/postinst

If we change, we should change all instances.

Reply:
Yes, please lets' change all comments and names stating "MSFT" which are not really appropriate for our public repro. Instead use "Core Team" or "OsConfig Core Team" as referenced in out documents in this repo. Thanks!